src/test/ui/feature-gates/feature-gate-raw-dylib.rs

```diff
@@ -1,5 +1,4 @@
-// gate-test-raw_dylib
```

Comment:
Shouldn't we keep the `gate-test` directive?

Reply:
Because the file's name is `feature-gate-$FEATURE.rs`, the test system automatically recognizes it as a test of that feature gate -- and in fact some tooling or other (`x.py test tidy`, maybe?) complains about the superfluous directive.  It's been a couple of weeks at this point, but I seem to remember that the first version of this PR actually failed automation because of the unnecessary directive.